setup.py

```diff
@@ -70,7 +70,10 @@
                      'dlkeras': [
                          'keras',
                          'tensorflow'
-                     ]
+                     ],
+                     'dltorch': [
+                         'torch==1.0.1',
```

Comment:
Does it support only this version of torch?

Reply:
1.1 also works perfectly fine but `mypy` fails on that one since they've fucked up type support. See https://github.com/pytorch/pytorch/issues/18724